Automattic/woocommerce-payments
# Reporting: Fix date filter on `View report` link on Disputes tile, within Payment activity widget

Fixes #8675 

This is a minor tweak that adds a date filter to the `View report` link on the Disputes tile. It adds a date filter to display disputes created in the last 7 days. It further helps add custom date ranges too for the link.


#### Testing instructions
* Check out this branch on your local client test site
* Make sure you have the feature flag enabled - `_wcpay_feature_payment_overview_widget`
* Browse to `Payments > Overview`
* Within the `Your payment activity` widget, mouseover `Disputes` and click on `View report` 
* It should lead to URL similar to `wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fdisputes&filter=advanced&date_between%5B0%5D=<date-one-week-ago>&date_between%5B1%5D=<todays-date>&status_is=needs_response`
* The filters on the page should be similar to the one in the screenshot, with the relevant dates:

<img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/d084a937-1777-47ed-9250-5de196350b49+" width="600px" />

-------------------


**Post merge**
